infrastructure/src/test/java/client/AccountsServiceApiTest.java

```diff
@@ -0,0 +1,32 @@
+package client;
+
+import generator.Generator;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import pojo.Account;
+
+import java.util.Optional;
+
+import static junit.framework.TestCase.assertNotNull;
+import static junit.framework.TestCase.assertTrue;
+
+public class AccountsServiceApiTest {
```

Comment:
As the comment for the production code, let's merge with `AccountsServiceClientTest`